report_sources/nhs_pathways_report_md_2020-05-13.Rmd

```diff
@@ -0,0 +1,272 @@
+description: We analyse publicly available data on self-reported COVID-19 symptoms
+tags: transmission-dynamics
+status: real-time-report
+update: 2020-05-16
+authors:
+  - id: quentin_leclerc
+    equal: 1
+  - id: emily_nightingale
+    equal: 1
+  - id: sam_abbott
+  - id: ncov-group
+  - id: thibaut_jombart
+---")
+
+```
+
+
+<!-- ======================================================= -->
+<!-- ======================================================= -->
+<!-- ======================================================= -->
+
+
+***This is a summary of the latest trends seen in potential COVID-19 cases reported through NHS Pathways. The results presented here include data up to `r database_date`. For a full discussion of this analysis, please refer to our paper available [here](link)***
+
+
+## Introduction
+
+NHS Pathways is a triage system for public calls and online reports for medical 
+care [@nhspathways]. This system is currently being used throughout England to assist 
+individuals reporting potential COVID-19 symptoms. Since the 18th of March 2020,
+data on daily phone calls and completed online assessments which have received
+a potential COVID-19 final disposition are openly available. These assessments
+are either completed via calls to 111 and 999 (which are respectively for
+non-urgent, and urgent medical problems), or through 111-online self-completed
+reports. The fraction of assessments corresponding to actual COVID-19 cases is
+unknown, but in the absence of wide-scale testing, the NHS Pathways dataset may
+be one of the best available proxies for COVID-19 incidence in the community.
+While prone to self-reporting biases, it is likely to better reflect milder
+cases and be less biased by different severity profiles than hospital admission
+data, which by definition reflect the most acute cases. 
+
+Here, we analyse NHS Pathways data until `r database_date` to assess the temporal
+dynamics of COVID-19 in England. Specifically, we investigated potential changes
+in the growth rate of the epidemic over time, and compared the observed patterns
+across NHS regions. We derived time-varying estimates of the growth rates,
+halving time and effective reproduction numbers for the different regions. We
+also assessed the potential correlation between NHS Pathways with COVID-19 daily
+deaths in England, to gain an initial understanding of its possible value within
+an early detection system.
+
+
+
+
+<!-- ======================================================= -->
+<!-- ======================================================= -->
+<!-- ======================================================= -->
+
+
+## Latest results
+
+***The results below were generated with data up to `r database_date`.***
+
+### Numbers of daily NHS Pathways report and NHS COVID-19 deaths
+
+<br>
+
+<img src="figures/nhs_paths/all_volume-1.png" width="70%" style="display: block; margin: auto;" />
```

Comment:
I don't see other files in the PR, just make sure all figures, bibliography files etc. are present.

Reply:
Yep that's fine, bib compiles here so not an issue, and for the figures folder I'll push this to the website manually